main.js

```diff
@@ -1392,11 +1392,11 @@ Inspector.prototype.registerMapTab = function () {
 					.range( [ 5, 18 ] );
 				map = L.map( 'map' ).setView( [ 0, 0 ], 2 );
 				new L.TileLayer(
-					'http://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png',
+					'//{s}.tile.openstreetmap.org/{z}/{x}/{y}.png',
 					{
 						minZoom: 2,
 						maxZoom: 18,
-						attribution: 'Map data © <a href="http://osm.org/copyright">OpenStreetMap</a> contributors'
+						attribution: 'Map data © <a href="//osm.org/copyright">OpenStreetMap</a> contributors'
```

Comment:
I get `ERR_CERT_COMMON_NAME_INVALID`. Maybe use `//www.openstreetmap.org/copyright` which seems to work?